docs/source/quickstart.rst

```diff
@@ -38,49 +39,70 @@ Dependencies
-  more with the `PySide <http://www.pyside.org/>`_ toolkit (LGPL license).
+* Either `wxPython <http://www.wxpython.org/>`_, `PyQt
+  <http://www.riverbankcomputing.co.uk/software/pyqt/intro>`_ (GPL or
+  Commercial license) or `PySide <http://www.pyside.org/>`_ (LGPL license) to
+  display interactive plots.
 
 Installing Chaco with Canopy
------------------------------
+----------------------------
 
 Chaco, the rest of the `Enthought Tool Suite <http://code.enthought.com/>`_,
-and a lot more are bundled with Canopy.  Getting Canopy allows you to install Chaco
+and a lot more are bundled with enthought Canopy (formerly EPD).  Getting
+Canopy gives you a one-click install of Chaco
 and all its dependencies at once; however, these packages will be linked to a
 new instance of Python. Canopy Express is free for all users and
 contains all that you need to use Chaco.
 
 To get Canopy, go to the `Canopy download page
-<http://www.enthought.com/products/canopy>`_ and get the appropriate
-version for your platform.  After running the installer, you will have a
-working version of Chaco and several examples.
+<http://store.enthought.com/>`_, select the desired bundle (Express, Full,
+TriplePlay) and get the appropriate version for your platform.  After running
+the installer, you will have a working version of Chaco and several examples.
 
 Building Chaco
 ---------------
 
 Building Chaco on your machine requires you to build Chaco and each of its
 dependencies, but it has the advantage of installing Chaco on top of the Python
 instance you already have installed.  The build process may be challenging and
-will require you to have SWIG, Cython and several development libraries
-installed.
+will require you to have a C compiler, SWIG, Cython and several development
+libraries installed.
 
-To do this, you can either
+To do this, you can either:
 
 1. Install Chaco and its :ref:`dependencies` from `PyPI
-   <http://pypi.python.org/pypi>`_ using `pip <http://www.pip-installer.org/en/latest/>`_. For example
+   <http://pypi.python.org/pypi>`_ using
+   `pip <http://www.pip-installer.org/en/latest/>`_ or using `easy_install
+   <http://packages.python.org/distribute/easy_install.html>`_ (part of
+   setuptools). For example
 
    :command:`pip install chaco`
 
+   or
+
+   :command:`easy_install chaco`
+
+.. note::
+   If you have already installed Chaco and just want to update to the newest
+   version, use
+
+        :command:`pip install --upgrade chaco`
+
+.. note::
+   Because Chaco depends on Enable, which depends on PIL, certain plarforms
+   cannot install PIL from PyPI without these options:
+
+        :command:`pip install --allow-external PIL --allow-unverified PIL chaco`
+
 2. Or, download the source from the `Chaco GitHub repository
-   <https://github.com/enthought/chaco>`_ or alternatively as a part of `ETS
-   <http://code.enthought.com/source/>`_.
+   <https://github.com/enthought/chaco>`_ or alternatively as a part of
```

Comment:
Can you point this directly to https://github.com/enthought/ets instead of code.e.com?